docs/sources/installation/windows.md

```diff
@@ -5,7 +5,7 @@ keywords = ["grafana", "configuration", "documentation", "windows"]
 type = "docs"
 [menu.docs]
 parent = "installation"
-weight = 3
+weight = 400
```

Comment:
Line 49: Building what? This is a topic about installing, so where should this bit about building Grafana actually be?

Reply:
@torkelo , can you answer this?

Reply:
@ivanahuckova , can you answer this?

Reply:
My opinion - I think we shouldn't mix installation documentation with building from source documentation - and we should keep those 2 separate. Therefore I would remove this information from here. Also, I have found documentation related directly to [building from source](https://grafana.com/docs/project/building_from_source/), where is this exact information mentioned as well. However, I couldn't find this file in this repo - I think it makes sense to have it here and move it from the other location. 
What do you think? @torkelo @marcusolsson

Reply:
I think we should differentiate between people who just want to build the latest version of Grafana, and people who are looking to develop Grafana. The latter should be covered in the [Developer guide](https://github.com/grafana/grafana/blob/master/contribute/developer-guide.md), but I think we should also consider having a stripped down guide for just building the binary. 

At this point though, everything in [Building from source](https://grafana.com/docs/project/building_from_source/) is covered in the Developer guide, so maybe we should replace it with a reference to the Developer guide for now?

Reply:
I like that plan, @marcusolsson. We don't want to confuse the laypeople, and it matches the way we've been organizing the contribution docs in the repo. 

Regarding the specific paragraph in windows.md, is that captured somewhere else already as well? Is it safe to delete it from windows.md?

Reply:
I don't think it's mentioned elsewhere right now. Either we leave that part as is, or we move it to the Developer guide?

Reply:
Let's leave it here for now.